redaxo/src/addons/mediapool/pages/index.php

```diff
@@ -101,5 +101,15 @@
     <?php
 }
 
+// -------------- Minibar
+if (class_exists('rex_minibar')) {
+    $page = rex_be_controller::getCurrentPageObject();
+
+    if ($page && $page->isPopup()) {
```

Comment:
```suggestion
    if ($page && $page->isPopup() && rex_minibar::getInstance()->isActive() === null) {
```

damit wir hier nur eine entscheidung treffen, falls vorher nicht schon jemand aktiv true/false gesetzt hat

Reply:
Finde ich gut

Reply:
Funktioniert. Im project-AddOn dann:

```php
rex_extension::register('PACKAGES_INCLUDED', function () {
    if (rex_be_controller::getCurrentPagePart(1) == 'mediapool') {
        rex_minibar::getInstance()->setActive(true);
    }
});
```